### DISCUSSION TITLE
Error: Please set async to true on the svc:services-config element

### DISCUSSION THREAD

**Discussion by greedo18** [Xomega-Net/Xomega.Net4VS#21] [Q&A]
I get the error in the title when I try to build out the Model. How can I do what they are asking?

**Comment by XomegaNet** [comment]
Thanks for reaching out. I'd be happy to help you with any questions or issues.

In your .Model project you should have a global configuration file `global_config.xom`, which should have a `svc:services-config` element. You need to set the `async="true"` attribute on it as follows.
```xml
  <svc:services-config async="true"
                       cancellation="true"
                       namespace="MyProject6.Services.Common"
                       implNamespace="MyProject6.Services.Entities"
                       tsOutputPath="../MyProject6.Client.Spa/ServiceContracts/{Module/}{File}"/>
```

This will allow generating async operations in your services, which is needed when accessing your REST API from the client (Wasm or WPF). Usually, the `async` attribute should be set by default by your Xomega Solution wizard, unless you explicitly uncheck it.

Please let me know if this helps fix the issue, or if you have any additional questions or issues.

**Comment by greedo18** [comment]
thanks for the note!  It may be a simple issue, but I don't see a
configuration file in the Model project. I generated a new project as per
the website and chose the standard setting.  Is the global config file
somewhere else? I pasted a picture of the structure.[image: image.png]

**Reply by XomegaNet** [reply]
hmm, that's weird. I cannot see the picture you pasted, but it looks like your `global_config.xom` file somehow disappeared after you imported your model from a database. Could you have accidentally deleted that file? 

In any event, can you try the following steps to rectify this issue?

1. Create a new Xomega solution in a separate folder using the same solution name and configuration.
2. Copy the `global_config.xom` file from the new solution over to the root of the Model project in your current solution.
3. Include the `global_config.xom` file in the Model project of the current solution (Show All Files > Include In Project).

Let me know if that fixes the issue.

**Reply by XomegaNet** [reply]
hi @greedo18, I'm just checking back to see if these steps helped you fix the issues you were having. Please let me know if they did, or if you have any additional questions.
